scripts/pyvmomi_to_ruby/QueryTypes.py

```diff
@@ -1,63 +0,0 @@
-# ******* WARNING - AUTO GENERATED CODE - DO NOT EDIT *******
```

Comment:
Is there any reason to delete the file now? or rather why this file was not fetched from url earlier?

Reply:
It was fetched from url but we don't commit the .py files and I wasn't aware about this when I upgraded pyvmomi to earlier version